concrete/views/dialogs/page/add/compose.php

```diff
@@ -11,12 +11,11 @@
             <button type="button" data-dialog-action="cancel" class="btn btn-default pull-left"><?=t('Cancel')?></button>
             <div class="btn-group pull-right">
```

Comment:
There is a `.btn-group` here, should we put the "clock" button under the "Publish Page" button and not inside ?